### ISSUE TITLE
Typescript compiler errors

### ISSUE THREAD

**Issue by wanxe** [mcabs3/google-places-web#23]
If I update to the 1.0.0 version and run tsc I get this errors: (I'm using typescript@3.6.4)

```
node_modules/google-places-web/dist/google/Places.d.ts:4:24 - error TS2304: Cannot find name 'IGooglePlacesConfig'.

4     constructor(opts?: IGooglePlacesConfig);
                         ~~~~~~~~~~~~~~~~~~~

node_modules/google-places-web/dist/google/Places.d.ts:5:27 - error TS2304: Cannot find name 'GooglePlacesAutocompleteOpts'.

5     autocomplete: (opts?: GooglePlacesAutocompleteOpts | undefined) => Promise<GooglePlaceAutocompleteResponse>;
                            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/google-places-web/dist/google/Places.d.ts:5:80 - error TS2304: Cannot find name 'GooglePlaceAutocompleteResponse'.

5     autocomplete: (opts?: GooglePlacesAutocompleteOpts | undefined) => Promise<GooglePlaceAutocompleteResponse>;
                                                                                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/google-places-web/dist/google/Places.d.ts:6:32 - error TS2304: Cannot find name 'GooglePlacesQueryAutocompleteOpts'.

6     queryautocomplete: (opts?: GooglePlacesQueryAutocompleteOpts | undefined) => Promise<GooglePlaceQueryAutocompleteResponse>;
                                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/google-places-web/dist/google/Places.d.ts:6:90 - error TS2304: Cannot find name 'GooglePlaceQueryAutocompleteResponse'.

6     queryautocomplete: (opts?: GooglePlacesQueryAutocompleteOpts | undefined) => Promise<GooglePlaceQueryAutocompleteResponse>;
                                                                                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/google-places-web/dist/google/Places.d.ts:7:22 - error TS2304: Cannot find name 'GooglePlacesDetailsOpts'.

7     details: (opts?: GooglePlacesDetailsOpts | undefined) => Promise<GooglePlaceDetailsResponse>;
                       ~~~~~~~~~~~~~~~~~~~~~~~

node_modules/google-places-web/dist/google/Places.d.ts:7:70 - error TS2304: Cannot find name 'GooglePlaceDetailsResponse'.

7     details: (opts?: GooglePlacesDetailsOpts | undefined) => Promise<GooglePlaceDetailsResponse>;
                                                                       ~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/google-places-web/dist/google/Places.d.ts:8:27 - error TS2304: Cannot find name 'GooglePlacesNearbySearchOpts'.

8     nearbysearch: (opts?: GooglePlacesNearbySearchOpts | undefined) => Promise<GooglePlaceNearbySearchResponse>;
                            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/google-places-web/dist/google/Places.d.ts:8:80 - error TS2304: Cannot find name 'GooglePlaceNearbySearchResponse'.

8     nearbysearch: (opts?: GooglePlacesNearbySearchOpts | undefined) => Promise<GooglePlaceNearbySearchResponse>;
                                                                                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/google-places-web/dist/google/Places.d.ts:9:25 - error TS2304: Cannot find name 'GooglePlacesTextSearchOpts'.

9     textsearch: (opts?: GooglePlacesTextSearchOpts | undefined) => Promise<GooglePlaceTextSearchResponse>;
                          ~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/google-places-web/dist/google/Places.d.ts:9:76 - error TS2304: Cannot find name 'GooglePlaceTextSearchResponse'.

9     textsearch: (opts?: GooglePlacesTextSearchOpts | undefined) => Promise<GooglePlaceTextSearchResponse>;
                                                                             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/google-places-web/dist/google/Places.d.ts:10:26 - error TS2304: Cannot find name 'GooglePlacesOptions'.

10     radarsearch: (opts?: GooglePlacesOptions) => Promise<GooglePlaceBaseResponse>;
                            ~~~~~~~~~~~~~~~~~~~

node_modules/google-places-web/dist/google/Places.d.ts:10:58 - error TS2304: Cannot find name 'GooglePlaceBaseResponse'.

10     radarsearch: (opts?: GooglePlacesOptions) => Promise<GooglePlaceBaseResponse>;
```

but it is fixed if I return to your 0.3.0 version.

Why it can be?

**Comment by wanxe** [comment]
I think that the types are missed in your build for some reason:

`https://raw.githubusercontent.com/mcabs3/google-places-web/53d697f984cd64302b35467ae2f76cfd3a95c9c0/src/%40types/index.d.ts`

**Comment by mcabs3** [comment]
@wanxe I apologize for the delay, I will take a look

**Comment by mcabs3** [comment]
@wanxe still new to how to include custom `.d.ts` files, so it looks like things were getting missed. I just published a 1.0.1 that I was testing on my location machine and seems to be good to go. Please let me know if you have any issues with it.

**Comment by wanxe** [comment]
Gracias @mcabs3 . I hope to try it this week
